tfhe/src/high_level_api/compressed_ciphertext_list.rs

```diff
@@ -97,16 +97,16 @@ impl Tagged for CompressedCiphertextList {
     }
 }
 
-impl CompressedCiphertextList {
-    pub fn len(&self) -> usize {
+impl CiphertextList for CompressedCiphertextList {
```

Comment:
where is the trait used btw ?

Reply:
here: https://github.com/zama-ai/tfhe-rs/pull/1482/commits/71523277f4e25ff5e8ef75706e8f45a5148f09bf#diff-629705e126043a5a46affa3b1c72fb1bb635d620b3515a2a895dfec783ff4313L295

With this trait I can de-duplicate some code but I can do without if you think that's not worth having a trait.